### ISSUE TITLE
when a new release on pypi? (>0.9.6)

### ISSUE THREAD

**Issue by oberix** [martinrusev/imbox#233]
Hi, are you planning for a new release to pypi soon? This project is quite handy and I'm using it in a couple of projects of my own, but since the latest release (0.9.6) in 2018 many bugs have been fixed. It would be nice to be able to get it from pypi.

Thanks
Regards

**Comment by jerryliang122** [comment]
you can use poety to add the project

**Comment by oberix** [comment]
Yep, maybe I put the question the wrong way. The point is not much having it on pypi, it's not having to pin a random commit. A tag would be enough I guess.

Thanks
